esphome/components/emporia_vue/emporia_vue.cpp

```diff
@@ -8,114 +9,131 @@ namespace emporia_vue {
 
 static const char *const TAG = "emporia_vue";
 
-EmporiaVueComponent *global_emporia_vue_component;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
+EmporiaVueComponent *global_emporia_vue_component =
+    nullptr;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
 
 void EmporiaVueComponent::dump_config() {
   ESP_LOGCONFIG(TAG, "Emporia Vue");
   LOG_I2C_DEVICE(this);
-
-  // TODO: Log phases
-  for (PhaseConfig *phase : this->phases_) {
-    //LOG_SENSOR("  ", "Phase", phase);
+  ESP_LOGCONFIG(TAG, "  Sensor Poll Interval: %dms", this->sensor_poll_interval_);
+
+  for (auto *phase : this->phases_) {
+    std::string wire;
+    switch (phase->get_input_wire()) {
+      case PhaseInputWire::BLACK:
+        wire = "BLACK";
+        break;
+      case PhaseInputWire::RED:
+        wire = "RED";
+        break;
+      case PhaseInputWire::BLUE:
+        wire = "BLUE";
+        break;
+    }
+    ESP_LOGCONFIG(TAG, "  Phase Config");
+    ESP_LOGCONFIG(TAG, "    Wire: %s", wire.c_str());
+    ESP_LOGCONFIG(TAG, "    Calibration: %f", phase->get_calibration());
+    LOG_SENSOR("    ", "Voltage", phase->get_voltage_sensor());
   }
 
-
-  for (CTSensor *ct_sensor : this->ct_sensors_) {
+  for (auto *ct_sensor : this->ct_sensors_) {
     LOG_SENSOR("  ", "CT", ct_sensor);
-    // TODO: Log other details
+    ESP_LOGCONFIG(TAG, "    Phase Calibration: %f", ct_sensor->get_phase()->get_calibration());
+    ESP_LOGCONFIG(TAG, "    CT Port Index: %d", ct_sensor->get_input_port());
   }
 }
 
 void EmporiaVueComponent::setup() {
   global_emporia_vue_component = this;
 
-  this->i2c_data_queue_ = xQueueCreate(1, sizeof(EmporiaSensorData));
-  xTaskCreatePinnedToCore(&EmporiaVueComponent::i2c_request_task,
-                          "i2c_request_task",
-                          8192,
-                          nullptr,
-                          0,
-                          nullptr,
-                          1
-  );
+  this->i2c_data_queue_ = xQueueCreate(1, sizeof(SensorReading));
+  xTaskCreatePinnedToCore(&EmporiaVueComponent::i2c_request_task, "i2c_request_task", 4096, nullptr, 0, nullptr, 1);
 }
 
 void EmporiaVueComponent::i2c_request_task(void *pv) {
-  const uint32_t NULL_CHECKSUM = 4294967295;
-  TickType_t xLastWakeTime;
-  const TickType_t xDelay = 240 / portTICK_PERIOD_MS;
-  uint32_t last_checksum = NULL_CHECKSUM;
+  const TickType_t poll_interval = global_emporia_vue_component->get_sensor_poll_interval() / portTICK_PERIOD_MS;
```

Comment:
Kinda split on what naming convention to use; I think snake case is a little better though since it's suggested by the [style guide](https://esphome.io/guides/contributing.html#codebase-standards)

Reply:
We should follow the style guide, so that name LGTM